js/app.js

```diff
@@ -0,0 +1,70 @@
+var todoList = angular.module('ToDoList', [] );
+
+todoList.controller('ToDoListController', [ function() {
+
+  var self = this;
+  self.list = [];
+  self.tasks = false;
+  self.taskNumber = 0;
+
+  self.updateTotal = function() {
+    self.taskNumber = 0;
+    for (x = 0; x < self.list.length; x ++ ) {
+      item = self.list[x];
+      if (item.status == 'Active') {
+        self.taskNumber ++;
+      }
+    }
+  };
+
+  self.checkTask = function() {
+    for (x = 0; x < self.list.length; x ++ ) {
+      item = self.list[x];
+      if (item.task == self.newTask) {
+        throw Error('Task already added');
```

Comment:
Do you really want to throw an error here? Where is it rescued? Does the user get feedback on this error?